custom-connectors/AzureOpenAIService/apiDefinition.swagger.json

```diff
@@ -0,0 +1,657 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure OpenAI Service",
+    "description": "Azure OpenAI Service provides REST API access to OpenAI's powerful language models including the GPT-3, Codex and Embeddings model series. These models can be easily adapted to your specific task including but not limited to content generation, summarization, semantic search, and natural language to code translation.",
+    "version": "1.0",
+    "contact": {
+      "name": "Daniel Laskewitz",
+      "url": "https://aka.ms/laskewitz",
+      "email": "connectorsupport@laskewitz.io"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://azure.microsoft.com/en-us/products/cognitive-services/openai-service"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://privacy.microsoft.com/en-us/privacystatement"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI"
+    }
+  ],
+  "host": "openai.azure.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/openai/deployments/{deployment-id}/completions": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
@Laskewitz is it possible to use more descriptive here. you can use something like "OK", "Success", "200", etc.
Please fix all responses descriptions in the swagger file.

Reply:
Done and fixed.